gcc/rust/expand/rust-macro-builtins-asm.cc

```diff
@@ -535,9 +537,9 @@ parse_format_string (InlineAsmContext &inline_asm_ctx)
 
 tl::optional<AST::Fragment>
 MacroBuiltin::asm_handler (location_t invoc_locus, AST::MacroInvocData &invoc,
-			   bool is_global_asm)
+			   bool semicolon, bool is_global_asm)
 {
-  return parse_asm (invoc_locus, invoc, is_global_asm);
+  return parse_asm (invoc_locus, invoc, is_global_asm, semicolon);
```

Comment:
It would have been better if the parameter order has not been changed, this is unexpected.